app/assets/stylesheets/v3/layout/body-font-size.scss

```diff
@@ -0,0 +1,4 @@
+// 18px Body font size
+body {
+    font-size: $base-font-size*1.125;
+}
```

Comment:
can we rename this file to base.scss? that way we can throw other similar stuff in here. Or we could create a folder called base, and put body-font-size in there?